src/drawing/DrawingAnnotations.tsx

```diff
@@ -76,13 +104,22 @@ const DrawingAnnotations = (props: Props): JSX.Element => {
                                 }
                             </DrawingPathGroup>
                         ))}
-                    </g>
+                    </DrawingSVGGroup>
                 </DrawingSVG>
             )}
 
             {isCreating && (
                 <DrawingCreator className="ba-DrawingAnnotations-creator" onStart={handleStart} onStop={handleStop} />
             )}
+
+            {isCreating && hasDrawnPathGroups && drawingSVGGroup && status !== CreatorStatus.staged && (
```

Comment:
The conditions in this area are getting a little unclear. There are actually two checks of `status` here since `isCreating` also includes a check for `pending`, but that's not immediately clear from reading this line. Can we simplify the logic?